_data/en/general.yml

```diff
@@ -1,3 +1,9 @@
+header:
+  tagline:             "Play music online. With friends. For free."
+  setupLink:           "Setup"
+  documentationLink:   "Documentation"
```

Comment:
Didn't follow the thread. But do we really want to link the manual here?

Also how can we ensure that this also gets shown on mobile devices?

If we design mobile first, we can't hide important links on mobile.

Reply:
The manual link is sort-of a temporary link. @gilgongo and I discussed that we should probably make a documentation hub page which has the full list of wiki pages and kb articles etc. 

About the header, that's a good question - I have to think about that. We'd probably have to change the nav dropdown. If we do go ahead with the docs hub, then we could just have the three links on the dropdown. But then we'd also have to ask how the list of docs will be shown in the hub.

Reply:
Ok. Sort of unrelated, but https://youtu.be/bQ-JwnGM2-c?t=163 might give some insight on our website.

Reply:
Wasn't he the guy who initially reviewed Jamulus (and some others?) and concluded that it didn't work?

Reply:
I think so? It would be great to get into contact with him...